libs/analytics/src/gtm/CowAnalyticsGtm.ts

```
@@ -0,0 +1,208 @@
+export class CowAnalyticsGtm implements CowAnalytics {
+  private dimensions: Record<string, string> = {}
+  private debouncedPageView = debounce((path?: string, params?: string[], title?: string) => {
+    this._sendPageView(path, params, title)
+  }, 1000)
+  private dataLayer: DataLayer = []
+
+  constructor() {
+    if (typeof window !== 'undefined') {
+      // Implement singleton pattern
+      if (window.cowAnalyticsInstance) {
+        return window.cowAnalyticsInstance
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)